GrandCentralBoard/Widgets/Bonus/Source/BonusSource.swift

```diff
@@ -104,7 +104,7 @@ final class BonusSource : Asynchronous {
     private func fetchImage(person: Person, closure: (ImageResultType) -> Void) {
         
         if let path = person.bubbleImage.url {
-            dataDownloader.downloadDataAtPath(path) { result in
+            downloader.requestData(.GET, url: NSURL(string: path)!, parameters: nil, headers: nil) { result in
```

Comment:
I am afraid about IUO NSURLs here. I think they could be (if they are not yet) `Decodable` and could throw exceptions when parsing configuration file.